test/system/252-quadlet.bats

```diff
@@ -558,4 +558,26 @@ EOF
     remove_secret $SECRET_NAME
 }
 
+@test "quadlet - volume path using specifier" {
+    local tmp_dir=specifier_path_$(random_string)
+    local file_name=hello.txt
+    local file_content="hello"
+    mkdir /tmp/$tmp_dir
```

Comment:
Please use `mktemp -d` or `$PODMAN_TMPDIR`.  `/tmp` may or may not work depending on the host.

Reply:
The reason I used `/tmp` directly was because I need `%T` to point to it. Otherwise it would not work. Having said that maybe `mktemp -d` would guaranty it as well. I'll try it out

Reply:
Done